spec/workers/step_by_step_draft_update_worker_spec.rb

```diff
@@ -25,16 +25,16 @@
     context 'when a guide is a new draft' do
       it 'should assign the draft to the current user' do
         described_class.new.perform(step_by_step_page.id, @current_user.name)
-        expect(StepByStepPage.find(step_by_step_page.id).assigned_to).to eql "New author"
+        expect(StepByStepPage.find(step_by_step_page.id).assigned_to).to include "New author"
       end
     end
     context 'when a guide is a draft that has already been updated' do
       before do
         step_by_step_page.mark_draft_updated
       end
-      it 'should remain assigned to the old author' do
+      it 'should be assigned to the new author' do
```

Comment:
I think what you are trying to test here is that `assigned_to` field hasn't been updated.

Rails has a nifty way to check for changed attributes. Basically, `assigned_to_changed?` should be false.

There's a better write up here: https://jasoncharnes.com/changed-attributes-rails